api/config/dex/config_request.go

```diff
@@ -30,6 +30,7 @@ func NewConfigRequest() *ConfigRequest {
 				Disclosure:           &ConfigRequest_V1_Disclosure{},
 				Banner:               &ConfigRequest_V1_Banner{},
 				InvalidLoginAttempts: &ConfigRequest_V1_InvalidLoginAttempts{},
+				Redirection:          &ConfigRequest_V1_Redirection{},
```

Comment:
Rename Redirection to RedirectionURIs in all the files.

Reply:
okay